src/index.html

```diff
@@ -4,9 +4,284 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.css">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="header__logo">
+        <img src="images/logo.png" alt="moyo logo">
+      </a>
+      <nav>
+        <ul class="nav">
+          <li class="nav__list">
+            <a href="#" class="nav__link nav__link--active">Apple</a>
+          </li>
+          <li class="nav__list">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__list">
+            <a href="#" class="nav__link">Смартфоны и телефоны</a>
+          </li>
+          <li class="nav__list">
+            <a href="#" class="nav__link" data-qa="nav-hover">Ноутбуки и компьютеры</a>
+          </li>
+          <li class="nav__list">
+            <a href="#" class="nav__link">Гаджеты</a>
+          </li>
+          <li class="nav__list">
+            <a href="#" class="nav__link">Планшеты</a>
+          </li>
+          <li class="nav__list">
+            <a href="#" class="nav__link">Фото</a>
+          </li>
+          <li class="nav__list">
+            <a href="#" class="nav__link">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="card">
+      <div class="card__wrapper" data-qa="card">
+        <div>
+          <img src="images/imac.jpeg" alt="apple laptop" class="card__image">
+        </div>
+        <div class="card__title">
+          <h4 class="card__header">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h4>
+          <p class="card__code">Код товара: 195434</p>
+        </div>
+        <div class="card__reviews">
+          <div class="card__stars">
+            <div class="card__star card__star--active"></div>
+            <div class="card__star card__star--active"></div>
+            <div class="card__star card__star--active"></div>
+            <div class="card__star card__star--active"></div>
+            <div class="card__star card__star--active"></div>
+          </div>
+          <p class="card__feedback">Отзывов: 5</p>
+        </div>
+        <div class="card__cost">
+          <p class="card__price">
+            Цена:
+          </p>
+          <p class="card__sum">
```

Comment:
це елемент від блоку ціни
```suggestion
          <p class="card-price__<name>">
```

Reply:
@gestap це його сусід він від блоку кост